test/validation/api/ipsec/ipsec_test_in.c

```diff
@@ -208,6 +208,38 @@ static void test_in_ipv4_esp_aes_cbc_null(void)
 	ipsec_sa_destroy(sa);
 }
 
+static void test_in_ipv4_esp_aes_cbc_sha1(void)
+{
+	odp_ipsec_sa_param_t param;
+	odp_ipsec_sa_t sa;
+
+	ipsec_sa_param_fill(&param,
+			    true, false, 123, NULL,
+			    ODP_CIPHER_ALG_AES_CBC, &key_a5_128,
+			    ODP_AUTH_ALG_SHA1_HMAC, &key_5a_160,
+			    NULL, NULL);
+
+	sa = odp_ipsec_sa_create(&param);
+
+	CU_ASSERT_NOT_EQUAL_FATAL(ODP_IPSEC_SA_INVALID, sa);
+
+	ipsec_test_part test = {
+		.pkt_in = &pkt_ipv4_icmp_0_esp_aes_cbc_sha1_1,
+		.num_pkt = 1,
+		.out = {
```

Comment:
This should be .in instead of .out now that the stats test addition changed things here. And the same applies to the other added tests.

Reply:
Ack. Updated to .in for all the newly added AES-CBC+SHAx inbound tests in v2.